lib/vorpal/config_builder.rb

```diff
@@ -90,7 +90,7 @@ def build_db_class
       # Module#parent comes from 'active_support/core_ext/module/introspection'
       parent_module = @domain_class.parent
 
-      return parent_module.const_get(db_class_name) if parent_module.const_defined?(db_class_name)
+      return parent_module.const_get(db_class_name) if Object.const_defined?(fully_qualified_db_class_name)
```

Comment:
Good question. The code was doing something like `AModule.const_defined?("PostDB")` and this changed it to `Object.const_defined?("AModule::PostDB")` since now we don't want to start the search from the `AModule` namespace, but rather from the top level namesapce.